spec/helpers/application_helper_spec.rb

```diff
@@ -6,6 +6,8 @@
     let(:bold) { '*bold*' }
 
     it 'renders markdown into html' do
+      allow(Flipflop).to receive(:treat_some_user_inputs_as_markdown?).and_return(true)
```

Comment:
Could you add a context for when treat_some_user_inputs_as_markdown? is set to false?

Reply:
Good idea, I can do that!